extensions/github/src/components/WorkflowActions.tsx

```diff
@@ -0,0 +1,134 @@
+import { Action, ActionPanel, Icon, showToast, Toast } from "@raycast/api";
+import { MutatePromise } from "@raycast/utils";
+import { Octokit } from "octokit";
+
+import { getErrorMessage } from "../helpers/errors";
+import { addFavoriteWorkflow, isFavoriteWorkflow } from "../helpers/favorite-workflows";
+import { getGitHubClient } from "../helpers/withGithubClient";
+import { WorkflowsResponse } from "../workflows";
+
+import { WorkflowForm } from "./WorkflowForm";
+
+export type Workflow = WorkflowsResponse["data"]["workflows"][0];
+
+export function getLastPathComponent(url: string): string {
+  return url.split("/").slice(-1)[0];
+}
+
+export function createWorkflowURL(workflow: Workflow): string {
+  const ymlFile = getLastPathComponent(workflow.path);
+  const path = getURLPathExceptLastPathComponent(workflow.url);
+  const url = replaceStringWithStringInPath(path, "https://api.github.com/repos/", "https://github.com/");
+  const workflowURL = addPathComponentToURL(url, ymlFile);
+  return workflowURL;
+}
+
+export async function runWorkflow(
+  workflow: Workflow,
+  repository: string,
+  branch: string,
+  inputs: any,
+  octokit: Octokit
```

Comment:
`octokit` can directly be imported into the function using `getGitHubClient`. Is there a reason why it's not the case here?